### PR TITLE
added jenkins app documentation

### PR DESCRIPTION
**Pull request by amr-yassir**
added jenkins application documentation to the manual

Related issues:
- codescalersinternships/home/issues/245

### CODE REVIEW

File: src/documentation/dashboard/solutions/jenkins.md
Comment (zaelgohary) [inline src/documentation/dashboard/solutions/jenkins.md:39]: Add planetary & IPv6 too.

**Review by zaelgohary — APPROVED** [review]

**Comment by Mik-TF** [comment]
@zaelgohary It's OK that we merged, but just a couple observations to take into account for future PRs.

Note the proposition above: `Planetary Network` to connect the Virtual Machine to Planetary network. This sentence isn't OK. If we wanted to keep the structure, we would at least need to write: the virtual machine to the Planetary network. We could have accepted this proposition before merging.

The app is in the left TOC of the manual, but not on the page Applications. Also note that the apps are in alphabetical order, so we would need to move Jenkins in the TOC.

The Jenkins app isn't yet available on Dashboard devnet, so usually we do not merge changes until it's on dashboard. Now the Jenkins app guide can be seen on manual.dev.grid.tf, but it isn't yet on dashboard.dev.grid.tf. If devnet is being updated soon, it's not really an issue.

Nothing major, just some details to make sure the workflow is OK. Thanks!

**Comment by Mik-TF** [comment]
Sorry if there was any confusion @zaelgohary. My suggestion above was in a pending state! Have a good day :)

**Pull request merged**
